test-data/samples/crawl2.py

```diff
@@ -0,0 +1,863 @@
+#!/usr/bin/env python3.4
+
+"""A simple web crawler."""
+
+# This is cloned from <asyncio>/examples/crawl.py,
+# with type annotations added (PEP 484).
+#
+# This version (crawl2.) has also been converted to use `async def` +
+# `await` (PEP 492).
```

Comment:
Hm, PEP 492 seems inconclusive about this. It certainly is harmless, but it appears that the `@asyncio.coroutine` decorator on an `async def` function is a no-op, so I'll remove them.